spec/services/reporting/total_user_count_report_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Reporting::TotalUserCountReport do
+  let(:report_date) { Time.zone.today + 1.day }
+  let!(:user) { create(:user) }
+  let(:expected_report) do
+    [
+      ['All-time user count'],
+      [expected_count],
+    ]
+  end
+
+  subject { described_class.new(report_date) }
+
+  describe '#total_user_count_report' do
+    let(:expected_count) { 1 }
+
+    it 'returns the expected data' do
+      expect(subject.total_user_count_report).to eq(expected_report)
+    end
+
+    context 'with a suspended user' do
+      let(:suspended_user) { create(:user, :suspended) }
+      let(:expected_count) { 2 }
+
+      it 'includes the suspended user in the count' do
+        expect(suspended_user).to be_suspended
```

Comment:
Aside: I just recently learned about how much magic you can do with built-in rspec matchers. If a class responds to `foo?`, then you can just `expect(subject).to be_foo`. https://rubydoc.info/gems/rspec-expectations/RSpec/Matchers#be-instance_method

This was kind of scaffolding for myself to confirm that my 'suspended' user was indeed suspended, but I'm inclined to leave it in for clarity.